addon/utils/stripe-mock.js

```diff
@@ -23,7 +23,7 @@ StripeMock.prototype.retrievePaymentIntent = function() {};
 StripeMock.prototype.handleCardPayment = function() {};
 StripeMock.prototype.handleCardAction = function() {};
 StripeMock.prototype.confirmPaymentIntent = function() {};
-StripeMock.prototype.handleCardSetup = function() {};
+StripeMock.prototype.confirmCardSetup = function() {};
```

Comment:
I think the test is failing because you've removed `handleCardSetup` here - can you put it back in?

Reply:
Ooops, sorry I missed that 🙏 . Weird that the other versions of ember passed...